test/e2e/quadlet_test.go

```diff
@@ -511,7 +526,7 @@ var _ = Describe("quadlet system generator", func() {
 	})
 
 	DescribeTable("Running quadlet test case",
-		func(fileName string) {
+		func(fileName string, exitCode int, errString string) {
```

Comment:
I think it will be cleaner to move the actual implementation of this function to an external function and then split between the test cases into two tables. The first will be remain as before and `func` will pass hardcoded `0 ""`, while the second will require the exit code and string from the entries.